src/main.rs

```diff
@@ -269,6 +269,8 @@ macro_rules! eat {
         }
     };
 }
+
+/// This macro search `token` in a line and returns it, or , else `continue` loop iteration.
```

Comment:
Is this what is really happening? It might be used like you wrote, but in the end this macro takes any expression which evaluates to a `Result`, returns the `Ok` value, or `continue`s in case of an `Err`.

Reply:
I can change to what you wrote instead.